scripts/script_dna_rna_tools.py

```diff
@@ -0,0 +1,72 @@
+VALID_DNA = set("ATGCatgc")
```

Comment:
```suggestion
valid_dna_set = set("ATGCatgc")
```
Капслок лишний.
-1 балл за нейминги не по pep8